pulp_rpm/tests/functional/constants.py

```diff
@@ -55,6 +49,20 @@
 }
 """The metadata for one RPM package."""
 
+RPM_PACKAGE_NAME = '{}'.format(RPM_PACKAGE_DATA['name'])
+"""The name of one RPM package."""
+
+RPM_PACKAGE_FILENAME = '{}-{}{}-{}.{}.rpm'.format(
+    RPM_PACKAGE_DATA['name'],
+    RPM_PACKAGE_DATA['epoch'] if RPM_PACKAGE_DATA['epoch'] != '0' else '',
```

Comment:
The impression I was getting is that even though the "package name", the NEVRA, includes Epoch information, the actual *filename* never does.

Reply:
I think you are right. But for this scenario there is no need to have more complex logic around the epoch, since it was hardcode as `0`.

Reply:
I'm not arguing for more complex logic, I'm saying I think we can just delete this line entirely, and simplify the format string as well.